BossMod/Modules/Endwalker/TreasureHunt/GymnasiouPithekos.cs

```diff
@@ -0,0 +1,126 @@
+using System.IO.Compression;
+using System.Security.Cryptography.X509Certificates;
+using FFXIVClientStructs.FFXIV.Common.Lua;
+
+namespace BossMod.Endwalker.TreasureHunt.GymnasiouPithekos
+{
+    public enum OID : uint
+    {
+        Boss = 0x3D2B, //R=6
+        BallOfLevin = 0x3E90,
+        BossAdd = 0x3D2C, //R=4.2
+        BossHelper = 0x233C,
+    };
+
+public enum AID : uint
+{
+    Attack = 872, // Boss->player, no cast, single-target
+    Thundercall = 32212, // Boss->location, 2,5s cast, range 3 circle
+    LightningBolt = 32214, // Boss->self, 3,0s cast, single-target
+    LightningBolt2 = 32215, // BossHelper->location, 3,0s cast, range 6 circle
+    ThunderIV = 32213, // BallOfLevin->self, 7,0s cast, range 18 circle
+    Spark = 32216, // Boss->self, 4,0s cast, range 14-30 donut --> TODO: confirm inner circle size
+    Attack2 = 870, // BossMikros->player, no cast, single-target
```

Comment:
What's BossMikros?

Also by convention autoattacks are called AutoAttack

Reply:
The adds are called Gymnasiou Pithekos Mikros, I couldn't think of a better name